src/devices_models/devices/common/constraints_structs.jl

```diff
@@ -1,12 +1,19 @@
 """ Data Container to construct range constraints"""
 struct DeviceRange
+    name::String
     limits::MinMax
     additional_terms_ub::Vector{Symbol}
     additional_terms_lb::Vector{Symbol}
 end
 
+function DeviceRange(name::String, limits::MinMax)
+    return DeviceRange(name, limits, Vector{Symbol}(), Vector{Symbol}())
+end
+
 struct DeviceTimeSeries
+    name::String
```

Comment:
Looks good to me but I slightly prefer having a single struct. @daniel-thom this will store the name twice, is that ok with you?